src/environment.ts

```diff
@@ -309,6 +309,8 @@ export class Environment {
     } else if (feature === 'WEBGL_CPU_FORWARD') {
       return true;
     } else if (feature === 'WEBGL_PACK') {
+      // Keep WEBGL_PACK default to false. You can turn on this feature by URL:
+      // http://localhost:1234/?tfjsflags=WEBGL_PACK:true
       return false;
```

Comment:
Can you restore it to the original code to keep same with upstream? By default, WEBGL_PACK will be true.

Reply:
Done, PTAL.